package.json

```diff
@@ -0,0 +1,72 @@
+{
+  "name": "mercurius-federation-info",
+  "version": "0.0.1",
+  "description": "A plugin for mercurius federation",
+  "main": "index.js",
+  "types": "index.d.ts",
+  "files": [
+    "index.js",
+    "index.d.ts",
+    "lib"
+  ],
+  "scripts": {
+    "lint": "eslint .",
+    "example": "node example/index.js",
+    "test": "npm run test:unit",
+    "test:unit": "tap --jobs=1 --coverage-report=html test/gateway/*.js",
+    "test:types": "tsd",
+    "prepare": "husky install"
+  },
+  "repository": {
+    "type": "git",
+    "url": "git+https://github.com/mercurius-js/federation-support"
+  },
+  "keywords": [
+    "mercurius"
+  ],
+  "author": "Davide Fiorello <davide@codeflyer.com>",
+  "license": "MIT",
+  "bugs": {
+    "url": "https://github.com/mercurius-js/federation-support/issues"
+  },
+  "homepage": "https://github.com/mercurius-js/federation-support#readme",
+  "devDependencies": {
+    "@commitlint/cli": "^17.1.2",
+    "@commitlint/config-conventional": "^17.1.0",
+    "@fastify/autoload": "5.4.0",
+    "@fastify/cors": "8.1.0",
+    "@sinonjs/fake-timers": "9.1.2",
+    "c8": "^7.12.0",
+    "desm": "^1.3.0",
+    "eslint": "^8.25.0",
+    "eslint-config-prettier": "^8.5.0",
+    "eslint-plugin-prettier": "^4.2.1",
+    "fastify": "^4.8.1",
+    "husky": "^8.0.1",
+    "lint-staged": "^13.0.3",
+    "mercurius": "11.1.0",
+    "prettier": "^2.7.1",
+    "sinon": "^14.0.0",
+    "tap": "^16.0.0",
+    "tsd": "^0.24.1"
+  },
+  "tsd": {
+    "directory": "test/types"
+  },
+  "lint-staged": {
+    "*.{js,jsx}": "eslint --cache --fix"
+  },
+  "peerDependencies": {
+    "fastify": "4.x",
+    "mercurius": "11.x"
+  },
```

Comment:
Avoid peerDependencies. They do not work the same across package managers :/.

Reply:
Removed